### PR TITLE
[New] dogfact command

### PR DESCRIPTION
**Pull request by Blazeflack**
It works the same as the catfact command.

### CODE REVIEW

File: Commands/Public/dogfact.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+						description: arrayFacts[i]
+					}
+				});
+			}
+		} else {
+			winston.error("Failed to fetch cat fact(s)", {svrid: msg.channel.guild.id, chid: msg.channel.id, usrid: msg.author.id});
+			msg.channel.createMessage({
+				embed: {
+                    color: 0x9ECDF2,
+					description: "Cats exist and are cute af. 😻"
```
Comment (KingDGrizzle) [inline Commands/Public/dogfact.js]: Blasphemy! Cats exist, but this is dogfacts

**Comment by KingDGrizzle** [comment]
This finally shows that dogs and cats are equal in this bot 😉

File: Commands/Public/dogfact.js (hunk 1 above)
Comment (Blazeflack) [inline Commands/Public/dogfact.js]: Woops, copy/paste fail :P

File: Commands/Public/dogfact.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ module.exports = (bot, db, config, winston, userDocument, serverDocument, channe
 			winston.error("Failed to fetch cat fact(s)", {svrid: msg.channel.guild.id, chid: msg.channel.id, usrid: msg.author.id});
```
Comment (KingDGrizzle) [inline Commands/Public/dogfact.js]: REEEEEEEEEEEEEEEEE, MORE CATS IN DOGS PALCE!

Comment (Blazeflack) [inline Commands/Public/dogfact.js]: Fixed, sorry :P

File: Configuration/commands.json [hunk 3]
```diff
@@ -245,6 +245,16 @@
             },
             "category": "Fun 🎪"
         },
+        "dogfact": {
+            "usage": "",
```
Comment (theHaden) [inline Configuration/commands.json]: The command appears to have usage other than blank. Change this to `[<fact-count>]` or similar.

File: Configuration/commands.json [hunk 4]
```diff
@@ -245,6 +245,16 @@
...
+            "description": "Random fact about dogs",
```
Comment (theHaden) [inline Configuration/commands.json]: The command allows for multiple facts — maybe this should be "Random fact(s) about dogs" instead?

File: Commands/Public/dogfact.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+						color: 0x00FF00,
+						description: arrayFacts[i]
+					}
+				});
+			}
+		} else {
+			winston.error("Failed to fetch dog fact(s)", {svrid: msg.channel.guild.id, chid: msg.channel.id, usrid: msg.author.id});
+			msg.channel.createMessage({
+				embed: {
+					color: 0x9ECDF2,
```
Comment (theHaden) [inline Commands/Public/dogfact.js]: Maybe use `0xFF3333` or `0xFF0000` to better signify there was an issue fetching the fact(s)

File: Commands/Public/dogfact.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+					description: "Dogs exist and are cute af. 😻"
+				}
```
Comment (theHaden) [inline Commands/Public/dogfact.js]: Maybe add a footer to better signify to the user that the fact(s) couldn't be fetched; e.g.

```js
embed: {
	text: "There was an error fetching the dog fact."
}
```

File: Commands/Public/dogfact.js (hunk 5 above)
Comment (KingDGrizzle) [inline Commands/Public/dogfact.js]: Failed is Red, aka `0xFF0000` I think

File: Commands/Public/dogfact.js (hunk 6 above)
Comment (KingDGrizzle) [inline Commands/Public/dogfact.js]: Already done on my side, but yeah

File: Configuration/commands.json (hunk 4 above)
Comment (KingDGrizzle) [inline Configuration/commands.json]: Same applies for Cat fact(s)

File: Commands/Public/dogfact.js (hunk 5 above)
Comment (theHaden) [inline Commands/Public/dogfact.js]: They're both red, the first one I suggested is just a different shade

**Comment by Blazeflack** [comment]
Considering the fact that @KingDGrizzle is re-doing a lot of the embeds, and apparently has implemented and improved the dogfact command locally, there is no point to keep this one going so I'm closing this pull request.

**Pull request closed without merging**